src/modules/LanItems.rb

```diff
@@ -2428,6 +2325,40 @@ def createS390Device
       result
     end
 
+    #  Creates a list of udev rules for old style named interfaces
+    #
+    #  It takes a whole "interfaces" section of AY profile. Keeps
+    #  only old style named interfaces (e.g. eth-bus-0.0.0000) in it
+    #  and renames them to a new name (e.g. eth0). It also produces
+    #  a list of udev rules to guarantee device naming persistency.
+    #  The rule is base on attributes described in old style name
+    #
+    #  @param [Array] list of hashes describing interfaces in AY profile
+    #  @return [Array] list of hashes for udev rules
+    def createUdevFromIfaceName(interfaces)
+      udev_rules = []
+      attr_map = {
+        "id"  => "ATTR{address}",
+        "bus" => "KERNELS"
+      }
+
+      interfaces.keep_if do |interface|
```

Comment:
well, it is described in the comment. It was the minimal change how to get rid of a global variable at the time of writing. I'll re-check if after other modifications it is possible to update it to more reasonable behavior.